mmv1/third_party/tgc_next/tfplan2cai/converters/cai/cai.go

```diff
@@ -0,0 +1,28 @@
+package cai
+
+import (
+	"regexp"
+
+	"github.com/hashicorp/terraform-provider-google-beta/google-beta/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google-beta/google-beta/transport"
+)
+
```

Comment:
Remove the structs from this file. Use the structs in [caiasset](https://github.com/GoogleCloudPlatform/terraform-google-conversion/blob/main/pkg/caiasset/asset.go)  instead.